### DISCUSSION TITLE
Ikea Parasoll device

### DISCUSSION THREAD

**Discussion by petzoom** [Koenkk/zigbee2mqtt#20461] [Devices]
Sorry for this question but I'm new with Zigbee2mqtt. I'd like to use the door sensor Parasoll from Ikea and I found on the forum article related to this device. It seems to be handled by the latest version of zigbee2mqtt, but, I've upgraded to the lastest docker image and the device is not recognized. Node red  return the mqtt message ; "Received message from unsupported device with Zigbee model 'PARASOLL Door/Window Sensor' and manufacturer name 'IKEA of Sweden'".
please, how to fix this problem ?

**Comment by Koenkk** [comment]
It will be supported in tomorrows release!

**Reply by petzoom** [reply]
Many thank's. I can't wait testing that.

**Reply by petzoom** [reply]
Ok it works fine, as I'm waiting for. Thx

**Comment by petzoom** [comment]
Hi, since the update to the release witch support thev ikea parasol door sensor, i have a big problème. The docker container of zigbee2mqtt restart every 5 min.
How to  check what s append? I made 2 différents update of the container without success.the log show some error , not very clear for me. 
[zigbee2mqtt (1).csv](https://github.com/Koenkk/zigbee2mqtt/files/14470934/zigbee2mqtt.1.csv)
Thank you for your help.

**Reply by petzoom** [reply]
Restart at 23h01, 23h06, 23h11,...

**Reply by petzoom** [reply]
Finally, I've solved my problem by removing devices in error in the web interface.
I'm not sure it is normal Z2M crash if a device failed. A message in the log would be better.
Regards
